index.js

```diff
@@ -64,9 +65,13 @@ export const AuthenticatedRoute = ({
 }) => {
 	const children = rest.children || null;
 
+	const {path, exact} = rest;
+
 	if (!isAuthenticated) {
-		// TODO: `history.location.pathname` probably doesn't work with nested routes
-		const redirect = to => to === history.location.pathname.replace(/\b\/.*$/, '') ? children : <Redirect to={to}/>;
+		const redirect = to => matchPath(to, {
+			path,
+			exact
+		}) ? children : <Redirect to={to}/>;
```

Comment:
You haven't tested the positive path (where the children render function is the one used). Do you mind adding one please? 😄

Reply:
PS: If you want to check whether you have tested all the branches or not, use `npx nyc --reporter=lcov ava`

Reply:
Just added that test, it was indeed missing 😄 
Regarding nyc, I didn't know that, seems great, although it still shows a 64.52% coverage since I only added tests for the AuthenticatedRoute component 😅 
Let me know if there's something else missing!